ehrenamt-justiz-backend/src/main/resources/application-local.yml

```
@@ -38,10 +38,8 @@ security:
-  muenchen:
-    ewo:
-      eai:
-        server: http://localhost:8085
-        user: testuser
-        password: testpw
\ No newline at end of file
+ewo:
+  eai:
+    server: http://localhost:8085
+    user: testuser
+    password: testpw
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)